src/views/_nodes_table.njk

```diff
@@ -0,0 +1,56 @@
+<table id="nodes-table">
+  <thead>
+      <tr>
+          <th>Software</th>
+          <th>Name</th>
+          <th>Version</th>
+          <th>Open signups</th>
+          <th>Total users</th>
+          <th>Active users half year</th>
+          <th>Active users monthly</th>
+          <th>Local posts</th>
+          <th>Local comments</th>
+          <th>Services</th>
+          <th>Country</th>
+      </tr>
+  </thead>
+  <tbody>
+      {% for pod in nodesData %}
+          {% set pod_meta = utils.get_pod_network_and_version(pod.network, pod.version) %}
+          {% if pod.name|lower == 'diaspora*' %}
+              {% set podname = pod.host %}
+          {% else %}
+              {% set podname = pod.name %}
+          {% endif %}
+          <tr>
+              {% if pod.network == "unknown" %}
```

Comment:
This code is copy pasted from the old page, I didn't even look at it. Don't hesitate if you have other feedback, I should also review / refactor it I guess. Maybe not in this PR ;)